packages/core/package.json

```diff
@@ -31,7 +31,6 @@
     "moment": "^2.24.0",
     "node-fetch": "^2.3.0",
     "solidity-coverage": "^0.7.2",
-    "supertest": "^4.0.2",
```

Comment:
What was this?

Reply:
This was the package I was using to test serverless. as this serverless is now in it's own package this is no longer needed in core :)